tests/Unit/Domain/CoordinateMaps/Test_DiscreteRotation.cpp

```diff
@@ -95,10 +96,16 @@ void test_3d() {
 void test_with_orientation() {
   INFO("With orientation");
   for (OrientationMapIterator<2> map_i{}; map_i; ++map_i) {
+    if (get(determinant(discrete_rotation_jacobian(*map_i))) < 0.0) {
```

Comment:
For my understanding: what is this (and similar `if (detJ < 0) {continue}` blocks below) checking for? Since you aren't allowing detJ < 0 in 2d or 3d, does this branch ever get taken?

Reply:
Yep, that's right. The iterator is over all, so this skips the "bad" orientations